helpers.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const readline = require('readline');
+const { appFolder } = require('./constants');
+const readStream = fs.createReadStream([appFolder, 'file.dat'].join("/"));
+
+exports.translateToGaro = (word) => {
+  const rl = readline.createInterface({
+    input: readStream,
+    crlfDelay: Infinity,
+  });
+  
+  rl.on('line', (line) => {
```

Comment:
I think we should add a mechanism of handling the case when a word is not found. It should throw an error saying that the word is not found. Was it actually saved?

Reply:
Added this.